ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -628,12 +628,14 @@
         event.stop();
         showDetails.hide();
         hideDetails.show();
+        $('similarity-button').removeClassName('collapsed');
```

Comment:
This code does nothing useful actually, since you're uncollapsing the button that is now hidden.